build.gradle.kts

```diff
@@ -52,3 +52,5 @@ kotlinter {
         "argument-list-wrapping",
     )
 }
+
+java.toolchain.languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
We don't use the toolchain feature in any other project either, the build should just use the user JVM.

Reply:
In this case with kotlin it makes sense to have it because of situations like this:
On my system I have Java 17, 19, 20 on the system where it defaults to another version you get an error saying 

Execution failed for task ':compileKotlin'.
> 'compileJava' task (current target is 20) and 'compileKotlin' task (current target is 17) jvm target compatibility should be set to the same Java version.
  Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain

So in cases like this it just makes sense to have this but if you want it removed I can remove it

Reply:
You can just set the java release
```kotlin
tasks {
  withType<JavaCompile> {
    options.release.set(17)
  }
}
```